auv_control.orogen

```diff
@@ -6,6 +6,7 @@ import_types_from "6dControl.hpp"
 using_library "motor_controller"
 
 typekit.export_types '/motor_controller/PIDSettings'
+typekit.export_types '/base/LinearAngular6DCommand'
```

Comment:
Hey @saarnold, I asked Sylvain before whether this was really necessary and he told me that `Typekits imports are not transitive.`, which I understood as imported types are not directly exported, so that we need to be explicity about it.

@doudou is this right?